page-objects/pages/TASKPage.js

```diff
@@ -167,13 +163,16 @@ async function clickCompletedTab(){
 	await elementHelpers.clickAndWait(completedTaskTab);
 }
 
+async function clickDraftsTab(){
```

Comment:
We should add this method as well:
async function clickOpenTasksTab(){
	await elementHelpers.clickAndWait(openTaskTab);
}

const openTaskTab= "//span[@data-testid = 'openTasks']";

Reply:
Will be done within BC-1008.